trunk/src/lsc/mysqldef.py

```diff
@@ -217,8 +217,10 @@ def ingestredu(imglist,force='no',dataredutable='photlco',filetype=1):
             print img,database
             lsc.mysqldef.deleteredufromarchive(string.split(img,'/')[-1],dataredutable)
             print 'delete line from '+str(database)
-            exist=lsc.mysqldef.getfromdataraw(conn,dataredutable,'filename', string.split(img,'/')[-1],column2='filename')
-
+            exist = ()
```

Comment:
This should be a boolean, not an empty tuple.

Reply:
but getfromdataraw give an empty tuple if the image is not there, so i though to be consistent with that. Anyway both are fine with me.